ssl/statem/extensions_clnt.c

```diff
@@ -1055,12 +1055,29 @@ int tls_parse_stoc_status_request(SSL *s, PACKET *pkt, unsigned int context,
          */
         if (chainidx != 0)
             return 1;
+
+        /*
+         * Ensure we get sensible values passed to tlsext_status_cb in the event
+         * that we don't receive a status message
```

Comment:
This doesn't make any sense because the "parse" functions only get called if the extension is present.